src/dialogs/ColumnPicker.js

```diff
@@ -22,8 +22,11 @@ var ColumnPicker = Dialog.extend('ColumnPicker', {
         this.grid = grid;
 
         if (behavior.isColumnReorderable()) {
-            // grab the lists from the behavior
-            if (behavior.setGroups){
+            // parse & add the drag-and-drop stylesheet addendum
+            var stylesheetAddendum = stylesheet.inject('list-dragon-addendum');
+
+            // grab the group lists from the behavior
+            if (behavior.setGroups) {
```

Comment:
working around what exactly? because if groups is `[]` shouldn't you be able to set them here? (I realize you are also setting `getCell` when you check "grouping" checkbox, so maybe we add that code here?)